examples/synthetic/2_synthetic_detect.py

```diff
@@ -0,0 +1,63 @@
+"""
+This script runs the detect stage for the synthetic example described in the tutorial
+in the online documentation. 
+
+:copyright:
+    2020–2024, QuakeMigrate developers.
+:license:
+    GNU General Public License, Version 3
+    (https://www.gnu.org/licenses/gpl-3.0.html)
+
+"""
+
+from quakemigrate import QuakeScan
+from quakemigrate.io import Archive, read_lut, read_stations
+from quakemigrate.signal.onsets import STALTAOnset
+
+
+# --- i/o paths ---
+station_file = "./inputs/synthetic_stations.sta"
+data_in = "./inputs/mSEED"
+lut_out = "./outputs/lut/example.LUT"
+run_path = "./outputs/runs"
+run_name = "example_run"
+
+# --- Set time period over which to run detect ---
+starttime = "2021-02-18T12:03:50.0"
+endtime = "2021-02-18T12:06:10.0"
+
+# --- Read in station file ---
+stations = read_stations(station_file)
+
+# --- Create new Archive and set path structure ---
+archive = Archive(
+    archive_path=data_in, stations=stations, archive_format="YEAR/JD/STATION"
+)
+
+# --- Load the LUT ---
+lut = read_lut(lut_file=lut_out)
+lut.decimate([4, 4, 1], inplace=True)
+
+# --- Create new Onset ---
+onset = STALTAOnset(position="centred", sampling_rate=50)
```

Comment:
I still generally use "classic" by default for detect, due to the much higher sensitivity of "centred" to data that contains sharp steps etc etc. As with many things this is based on gut feel and anecdotal experience (followed by good experience using "classic" - could of course be way better using "centred" and I just don't know about it..)

Reply:
Ah the accompanying docs on synthetic-locate do emphasise using "centred" instead of "classic", so I guess this was maybe not deliberate..

Reply:
Yes, likely just a copy-paste hangover. Will adjust.